internal/services/manager.go

```diff
@@ -194,14 +170,14 @@ func StartProcessManager(
 	if processComposeBackground {
 		flags = append(flags, "-t=false")
 		cmd := exec.Command(processComposeBinPath, flags...)
-		return runProcessManagerInBackground(cmd, port, config, projectDir)
+		return runProcessManagerInBackground(cmd, config, port, projectDir)
 	}
 
 	cmd := exec.Command(processComposeBinPath, flags...)
-	return runProcessManagerInForeground(cmd, port, projectDir, w)
+	return runProcessManagerInForeground(cmd, config, port, projectDir, w)
```

Comment:
this feels a bit problematic:
1. above we are doing `defer configFile.Close()`
2. Inside `runProcessManagerInForeground`, we are closing the file, waiting for the command, and re-opening a file. 

I worry that since the step (1) defer runs after step (2) it will error because it is closing a file that's already closed.

Oh, but we are implicitly discarding the error from `file.Close()` in the `defer` block, so all good I think!

Reply:
I guess another option is to defer the entire cleanup block after `cmd.Wait` call + error check in `runProcessManagerInForeground` to ensure that it runs -- but I agree that discarding the error from `defer configFile.Close()` avoids the issue as well.